app/api/currentuser/get_dump.feature

```diff
@@ -57,19 +57,21 @@ Feature: Export the short version of the current user's data
       | 12                | 1              | false   |
       | 12                | 2              | false   |
       | 12                | 12             | true    |
+    And the database has the following table 'items':
+      | id  |
+      | 404 |
     And the database has the following table 'users_answers':
       | id | user_id | item_id | submitted_at        |
       | 1  | 2       | 404     | 2019-07-09 20:02:28 |
       | 2  | 3       | 404     | 2019-07-09 20:02:28 |
-    And the database has the following table 'users_items':
-      | id | user_id | item_id |
-      | 11 | 2       | 404     |
-      | 12 | 3       | 404     |
     And the database has the following table 'groups_attempts':
       | id  | group_id | item_id | order |
       | 111 | 11       | 404     | 0     |
       | 112 | 2        | 404     | 0     |
       | 113 | 1        | 404     | 0     |
+    And the database has the following table 'users_items':
+      | user_id | item_id | active_attempt_id |
+      | 2       | 404     | 111               |
```

Comment:
I see in all these changes of features file that it is partially to define `users_items` after `groups_attempts`. 
I think that will quickly be boring to think about FK when writing such dataset, so probably there should be a `SET FOREIGN_KEY_CHECKS = 0` before setting all these tables (set to 1 when ? when doing the request 😕).

Reply:
By not doing this way, I can be sure that all the needed data are there

Reply:
Ah when `SET FOREIGN_KEY_CHECKS = 1`, it would not recheck everything? Make sense. There should be able to recheck everything. 
Anyway, when we'll cross FK (e.g. item have a default string, string are related to item), we'll have to do this.